gbfs.md

```diff
@@ -571,14 +572,90 @@ Describes pricing for the system.
+&emsp;&emsp;\-&nbsp;`start` | Optional | Non-Negative Integer | Number of units that have to elapse before this segment starts applying.<br /><br />If this field is empty, the price issued from this segment is charged immediately upon rental.
+&emsp;&emsp;\-&nbsp;`rate` | Yes | Float | Rate that serves to calculate the variable price based on the amount of variable unites. The rate is valid for the segment. Can be a negative number, which indicated that the traveller will receive a discount.
+&emsp;\-&nbsp;`flat_segments` | Conditionally Required | Array | Array of flat prices in increasing order. Each segment defines the flat price and the interval of `variable_unit` of riding in the pricing plan currency, in which the flat price is reapplied.<br /><br />At least on `rate_segment` or `flat_segment` is required.
+&emsp;&emsp;\-&nbsp;`price` | Yes | Float | Flat price charged per interval. Currency is defined by the `currency` field.
+&emsp;&emsp;\-&nbsp;`interval` | Yes | Non-Negative Integer | Interval in `variable_unit` at which the price of this segment is charged.
+&emsp;&emsp;\-&nbsp;`start` | Optional | Non-Negative Integer | Number of units that have to elapse before this segment starts applying.<br /><br />If this field is empty, the price issued from this segment is charged immediately upon rental.
+\-&nbsp;`surge_pricing` | Optional | Non-Negative Float | Multiplier that is applied during period of increased demand. If this field is empty, there is no surge pricing.
+
+Example:
+This example demonstrates a pricing scheme that has a rate both by minute and by km. The user is charged $0.25 per km as well as $0.50 per minute. Both of these rates happen concurrently and are not dependent on one another. 
+```jsonc
+{
+  "plans": {
+    "plan_id": "plan2",
+    "start_time": "7:00:00",
+    "end_time": "9:30:00",
+    "vehicle_type_id": "def",
+    "name": "Rush Hour Rate",
+    "currency": "CAD",
+    "price": 3,
+    "is_taxable": true,
+    "description": "$3 unlock fee, $0.25 per kilometer and $0.50 per minute",
+    "variable_price": [
+      {
+        "variable_unit": "km",
+        "rate_segments": {
+          "start": 0,
+          "rate": 0.25
+        }
+      },
+      {
+        "variable_unit": "minute",
+        "rate_segments": {
+          "start": 0,
+          "rate": 0.5
+        }
+      }
+    ]
+  }
+}
+```
+
+Example:
+
+The user does not pay more than the base price for the first 10 km. After 10 km the user pays $1 per km. After 25 km the user pays $0.50 per km and an additional $3 every 5 km, the extension price, in addition to $0.50 per km. 
+
+```jsonc
+ "variable_price": {
+   "variable_unit": "km",
+   "rate_segments": [
+     {
+       "start": 10,
+       "rate": 1
+     },
+     {
+       "start": 25,
+       "rate": 0.5
+     }
+   ]
+   "flat_segments": [
+     {
+       "start": 25,
+       "price": 3,
+       "interval": 5,
+     }
+   ]
+ }
```

Comment:
Per the description above `variable_price` is an array.

```suggestion
"variable_price": [
    {
        "variable_unit": "km",
        "rate_segments": [
            {
                "start": 10,
                "rate": 1
            },
            {
                "start": 25,
                "rate": 0.5
            }
        ],
        "flat_segments": [
            {
                "start": 25,
                "price": 3,
                "interval": 5
            }
        ]
    }
]
```

Reply:
Would this be functionally the same thing? `rate_segments` seems like a special case of `flat_segments` where the interval is 1.

```jsonc
"variable_price": [
    {
        "variable_unit": "km",
        "flat_segments": [
            {
                "start": 10,
                "price": 1,
                "interval": 1
            },
            {
                "start": 25,
                "price": 0.5,
                "interval": 1
            }
        ]
    },
    {
        "variable_unit": "km",
        "flat_segments": [
            {
                "start": 25,
                "price": 3,
                "interval": 5
            }
        ]
    }
]
```

Reply:
I've added this to the new round of changes, thanks for this suggestion!